### DISCUSSION TITLE
Modelando Domínios Ricos - Organizando a Solução - Como organizar no VisualStudio

### DISCUSSION THREAD

**Discussion by Arthurresendee** [balta-io/forum#605] [Tira Dúvidas] [answered]
Tô me sentindo uma anta aqui kkk. Como eu inicio criando meu projeto a partir de uma solução em branco no VisualStudio? Faço do mesmo jeito que no vscode? tentei aqui mas quando crio as pastas e depois a solução sai tudo bugado, fora de referência. Nem começou e eu já travei, se alguém aí pudar dar uma dica... Acredito que seja coisa boba. Quando crio algum projeto ele já cria a solução direto, agora criar pastas e depois a solução me travou.

**Comment by henriqueweege** [comment, marked as answer]
Fala, @Arthurresendee ! Tudo certo?
Viu, se vc criar tudo via CLI funciona mais fácil.
Tem como mandar print de como estão as coisas por aí? Assim fica mais fácil de te ajudar.
Valeeeu!

**Reply by Arthurresendee** [reply]
Criei tudo via CLI mesmo, deu certo aqui valeu.

**Reply by Arthurresendee** [reply]
Mas como funcionaria esse processo no VS mais ou menos? sabe me dizer?

**Reply by henriqueweege** [reply]
Boa!
É só você criar um novo projeto, e depois, clicar com o botão direito na solução e ir em Adicionar->Novo projeto. Vai abrir um tela para você escolher o modelo do projeto (eu escolhi "classlib"). Quando clicar em "Próximo", vai abrir uma outra janela para escolher o nome do projeto (eu deixei o que estava mesmo). Na próxima etapa, vai ser a hora de escolher a versão do .NET a ser usada (de novo eu deixei a que estava, mas é aconselhável utilizar a mesma versão do curso).
Depois disso, basta clicar em "Criar", e você vai ter um novo projeto adicionado na tua solução!

É isso! Valeeeu! 

![image](https://github.com/balta-io/forum/assets/96916801/f2cad070-c99d-456d-93ab-cc9fca5041ff)
![image](https://github.com/balta-io/forum/assets/96916801/ade1ed6a-a84b-4a79-8ef2-31229a7a0e28)
![image](https://github.com/balta-io/forum/assets/96916801/bb6a45ba-c17c-4d00-bf5e-2961354917b5)
![image](https://github.com/balta-io/forum/assets/96916801/d28ffc4c-9126-4888-8a0d-b63942322374)
![image](https://github.com/balta-io/forum/assets/96916801/d7e7df57-cf75-4e72-880e-ef6a7d78fb86)
![image](https://github.com/balta-io/forum/assets/96916801/142212fc-006c-4053-a863-94e25319169f)

**Comment by Arthurresendee** [comment]
Ahhh, pô era tão simples, eu é que estava me confundindo aqui. Obrigado mano!

**Reply by henriqueweege** [reply]
Tmj meu caro! Qualquer coisa só dar um toque!
Valeeeu!
